build/webpack/_development.js

```diff
@@ -41,12 +42,33 @@ export default (webpackConfig) => {
           imports   : ['react'],
           locals    : ['module']
         }
-        loader.query.env.development.plugins[0][1].transforms
-          .push(reactTransformHmr)
+
+        loader.query.env.development.plugins =
+          loader.query.env.development.plugins.map(plugin => {
```

Comment:
@justingreenberg definitely agree ^^, I hate the syntax, it makes configuration horrendously complicated.